app/javascript/chat/content.jsx

```diff
@@ -44,23 +44,20 @@ export class Content extends Component {
     if (!resource) {
       return '';
     }
-
     return (
       // TODO: A button (role="button") cannot contain other interactive elements, i.e. buttons.
       // TODO: These should have key click events as well.
       // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
nitpick (blocking) - we can delete these comments now as this fixes the issue they relate to 🙂